index/static/js/contestTimer.js

```diff
@@ -68,11 +68,21 @@ function getRestTime() {
             result /= 60;
             var m = parseInt(result % 60);
             result /= 60;
-            var h = parseInt(result);
+            var h = parseInt(result % 24);
+            result /= 24;
+            var d = parseInt(result);
             m = checkTime(m);
             s = checkTime(s);
             h = checkTime(h);
-            remainings[i].innerHTML = h + ":" + m + ":" + s;
+            var day;
```

Comment:
the function about checkTime is to add a '0' if the number < 10,and I think 01 day or 02 days would be weird.